plugins/GSdx/GSState.h

```diff
@@ -243,6 +243,7 @@ class GSState : public GSAlignedClass<32>
 	void ResetHandlers();
 
 	int GetFramebufferHeight();
+	void SaturateOutputSize(GSVector4i*);
```

Comment:
Any reason to pass by reference? As far as I can see, I only see a demerit that the function call wouldn't have ``&`` which was necessary to show that the variable's value might be modified by the function.